libs/ui-lib/lib/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -73,7 +73,9 @@ const ClusterDetailsForm = (props: ClusterDetailsFormProps) => {
   const handleSubmit = React.useCallback(
     async (values: OcmClusterDetailsValues) => {
       if (cluster) {
-        const params = ClusterDetailsService.getClusterUpdateParams(values);
+        const resetPlatform = cluster.userManagedNetworking ? 'none' : values.platform;
```

Comment:
I have some questions here:

1. I'm not sure if it's possible to have the combination of a platform integration + UMN. If it's possible, then this code would remove the platform integration.
2. For the case where we want to move from one integration to no integration, the code should use `none` or `baremetal`. Since the dropdown only has none, we would never be patching this to `baremetal` and we would get an error when trying to PATCH.

![patch-fail](https://github.com/openshift-assisted/assisted-installer-ui/assets/829045/579ffcc7-ca2d-4cb1-b145-cecea3406387)

Reply:
After talking with BE, this is the correct behavior:
![image](https://github.com/openshift-assisted/assisted-installer-ui/assets/11390125/0727e2ef-1774-4fd0-8a1c-908527c8b98a)

Reply:
From this thread: https://redhat-internal.slack.com/archives/CUPJTHQ5P/p1689581850055749

- OCI- UMN
- Vsphere-CMN
- Nutanix-CMN
- Baremetal-both